ts/BUILD.typescript

```diff
@@ -26,6 +29,8 @@ write_file(
     out = "tsc.js",
     content = [
         "const {dirname} = require('path')",
+        "const {link} = require('@rules_js/linker')",
```

Comment:
same here, soon we'll want to remove this JS file and point `nodejs_binary` directly at the file in external, so we want `--require` to install the linker